wave_matching/tests/ndt_tests.cpp

```diff
@@ -25,8 +25,9 @@ class NDTTest : public testing::Test {
         pcl::io::loadPCDFile(TEST_SCAN, *(this->ref));
     }
 
-    void initMatcher(const float res, const Affine3 perturb) {
-        this->matcher = new NDTMatcher(res, TEST_CONFIG);
+    void initMatcher(const NDTMatcherParams params,
+                     const Eigen::Affine3d perturb) {
+        this->matcher = new NDTMatcher(params);
```

Comment:
The constructor sets the configuration so I'm using a pointer so that the configuration can be set later.